src/AGTHARN/FlyPE/lists/ParticleList.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+/* 
+ *  ______ _  __     _______  ______ 
+ * |  ____| | \ \   / /  __ \|  ____|
+ * | |__  | |  \ \_/ /| |__) | |__   
+ * |  __| | |   \   / |  ___/|  __|  
+ * | |    | |____| |  | |    | |____ 
+ * |_|    |______|_|  |_|    |______|
+ *
+ * FlyPE, is an advanced fly plugin for PMMP.
+ * Copyright(C) 2020 AGTHARN
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ *(at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace AGTHARN\FlyPE\lists;
+
+use pocketmine\utils\TextFormat as C;
+use pocketmine\math\Vector3;
+
+use pocketmine\level\particle\BubbleParticle;
+use pocketmine\level\particle\CriticalParticle;
+use pocketmine\level\particle\EnchantParticle;
+use pocketmine\level\particle\InstantEnchantParticle;
+use pocketmine\level\particle\ExplodeParticle;
+use pocketmine\level\particle\HugeExplodeParticle;
+use pocketmine\level\particle\EntityFlameParticle;
+use pocketmine\level\particle\FlameParticle;
+use pocketmine\level\particle\HeartParticle;
+use pocketmine\level\particle\InkParticle;
+use pocketmine\level\particle\LavaDripParticle;
+use pocketmine\level\particle\LavaParticle;
+use pocketmine\level\particle\PortalParticle;
+use pocketmine\level\particle\RedstoneParticle;
+use pocketmine\level\particle\SmokeParticle;
+use pocketmine\level\particle\SplashParticle;
+use pocketmine\level\particle\SporeParticle;
+use pocketmine\level\particle\MobSpawnParticle;
+use pocketmine\level\particle\WaterDripParticle;
+use pocketmine\level\particle\WaterParticle;
+use pocketmine\level\particle\EnchantmentTableParticle;
+use pocketmine\level\particle\HappyVillagerParticle;
+use pocketmine\level\particle\AngryVillagerParticle;
+use pocketmine\level\particle\RainSplashParticle;
+
+use AGTHARN\FlyPE\Main;
+
+class ParticleList {
+
+    /**
+     * plugin
+     * 
+     * @var Main
+     */
+    private $plugin;
```

Comment:
Unused property, you can also remove the `$plugin` parameter in the constructor for this class, just make sure you change it in the areas where you create new `ParticleList` objects.